### PR TITLE
[zh-cn] update radicalgradient

### PR DESCRIPTION
**Pull request by jasonren0403**
### Related issues and pull requests

part of #12578

### CODE REVIEW

File: files/zh-cn/web/svg/element/radialgradient/index.md [hunk 1]
```diff
@@ -1,77 +1,84 @@
...
-- {{SVGAttr("externalResourcesRequired")}}
+{{EmbedLiveSample("示例", 120, 120)}}
 
-### 专有属性
+## 属性
 
-- {{SVGAttr("gradientUnits")}}
-- {{SVGAttr("gradientTransform")}}
 - {{SVGAttr("cx")}}
+  - : 这个属性定义了径向渐变的终点圆的 X 坐标。*值类型*：[**\<length>**](/zh-CN/docs/Web/SVG/Content_type#长度)；*默认值*：`50%`；*可使用动画*：**是**
```
Comment (yarusome) [inline files/zh-cn/web/svg/element/radialgradient/index.md:44]: animatable 名词化译为“动画性”，见 #12600 。

File: files/zh-cn/web/svg/element/radialgradient/index.md [hunk 2]
```diff
@@ -1,77 +1,84 @@
...
 - {{SVGAttr("cy")}}
```
Comment (jasonren0403) [inline files/zh-cn/web/svg/element/radialgradient/index.md:45]:
```suggestion
  - : 这个属性定义了径向渐变的终点圆的 X 坐标。*值类型*：[**\<length>**](/zh-CN/docs/Web/SVG/Content_type#长度)；*默认值*：`50%`；*动画性*：**有**
- {{SVGAttr("cy")}}
```

File: files/zh-cn/web/svg/element/radialgradient/index.md [hunk 3]
```diff
@@ -1,77 +1,84 @@
...
-- {{SVGAttr("r")}}
+  - : 这个属性定义了径向渐变的终点圆的 Y 坐标。*值类型*：[**\<length>**](/zh-CN/docs/Web/SVG/Content_type#长度)；*默认值*：`50%`；*可使用动画*：**是**
+- {{SVGAttr("fr")}}
+  - : 这个属性定义了径向梯度的起点圆的半径。渐变的绘制将使 0% {{SVGElement('stop','gradient stop')}} 被映射到起始圆的周长。*值类型*：[**\<length>**](/zh-CN/docs/Web/SVG/Content_type#长度)；*默认值*：`0%`；*可使用动画*：**是**
 - {{SVGAttr("fx")}}
+  - : 这个属性定义了径向渐变的起点圆的 X 坐标。*值类型*：[**\<length>**](/zh-CN/docs/Web/SVG/Content_type#长度)；*默认值*：与 `cx` 相同；*可使用动画*：**是**
```
Comment (jasonren0403) [inline files/zh-cn/web/svg/element/radialgradient/index.md:50]:
```suggestion
  - : 这个属性定义了径向渐变的终点圆的 Y 坐标。*值类型*：[**\<length>**](/zh-CN/docs/Web/SVG/Content_type#长度)；*默认值*：`50%`；*动画性*：**有**
- {{SVGAttr("fr")}}
  - : 这个属性定义了径向梯度的起点圆的半径。渐变的绘制将使 0% {{SVGElement('stop','gradient stop')}} 被映射到起始圆的周长。*值类型*：[**\<length>**](/zh-CN/docs/Web/SVG/Content_type#长度)；*默认值*：`0%`；*动画性*：**有**
- {{SVGAttr("fx")}}
  - : 这个属性定义了径向渐变的起点圆的 X 坐标。*值类型*：[**\<length>**](/zh-CN/docs/Web/SVG/Content_type#长度)；*默认值*：与 `cx` 相同；*动画性*：**有**
```

File: files/zh-cn/web/svg/element/radialgradient/index.md [hunk 4]
```diff
@@ -1,77 +1,84 @@
...
+- {{SVGAttr("gradientTransform")}}
+  - : 这个属性为梯度坐标系提供了额外的[变换](/zh-CN/docs/Web/SVG/Atribute/transform)。*值类型*：**[\<transform-list>](/zh-CN/docs/Web/SVG/Content_type#transform-list)** ；*默认值*：*identity transform*；*可使用动画*：**是**
+- {{SVGAttr("href")}}
+  - : 这个属性定义了对另一个将被用作模板 `<radialGradient>` 元素的引用。*值类型*：[**\<URL>**](/zh-CN/docs/Web/SVG/Content_type#url)；*默认值*：none；*可使用动画*：**是**
+- {{SVGAttr("r")}}
+  - : 这个属性定义了径向渐变的终点圆的半径。梯度的绘制将使 100% 的{{SVGElement('stop','gradient stop')}} 被映射到终点圆的周长。*值类型*：[**\<length>**](/zh-CN/docs/Web/SVG/Content_type#长度)；*默认值*：`50%`；*可使用动画*：**是**
 - {{SVGAttr("spreadMethod")}}
+  - : 这个属性表明，如果渐变在包含渐变的形状的边界内开始或结束，它将如何表现。*值类型*：`pad`|`reflect`|`repeat`；*默认值*：`pad`；*可使用动画*：**是**
 - {{SVGAttr("xlink:href")}}
+  - : {{Deprecated_Header}}对另一个将被用作模板的 `<radialGradient>` 元素的 [\<IRI>](/zh-CN/docs/Web/SVG/Content_type#iri) 引用。*值类型*：[**\<IRI>**](/zh-CN/docs/Web/SVG/Content_type#iri)；*默认值*：none；*可使用动画*：**是**
```
Comment (jasonren0403) [inline files/zh-cn/web/svg/element/radialgradient/index.md:64]:
```suggestion
  - : 这个属性定义了径向渐变的起点圆的 Y 坐标。*值类型*：[**\<length>**](/zh-CN/docs/Web/SVG/Content_type#长度)；*默认值*：与 `cy` 相同；*动画性*：**有**
- {{SVGAttr("gradientUnits")}}
  - : 这个属性定义了 `cx`、`cy`、`r`、`fx`、`fy`、`fr` 属性的坐标系统；*值类型*：`userSpaceOnUse`|`objectBoundingBox`；*默认值*：`objectBoundingBox`；*动画性*：**有**
- {{SVGAttr("gradientTransform")}}
  - : 这个属性为梯度坐标系提供了额外的[变换](/zh-CN/docs/Web/SVG/Atribute/transform)。*值类型*：**[\<transform-list>](/zh-CN/docs/Web/SVG/Content_type#transform-list)** ；*默认值*：*identity transform*；*动画性*：**有**
- {{SVGAttr("href")}}
  - : 这个属性定义了对另一个将被用作模板 `<radialGradient>` 元素的引用。*值类型*：[**\<URL>**](/zh-CN/docs/Web/SVG/Content_type#url)；*默认值*：none；*动画性*：**有**
- {{SVGAttr("r")}}
  - : 这个属性定义了径向渐变的终点圆的半径。梯度的绘制将使 100% 的{{SVGElement('stop','gradient stop')}} 被映射到终点圆的周长。*值类型*：[**\<length>**](/zh-CN/docs/Web/SVG/Content_type#长度)；*默认值*：`50%`；*动画性*：**有**
- {{SVGAttr("spreadMethod")}}
  - : 这个属性表明，如果渐变在包含渐变的形状的边界内开始或结束，它将如何表现。*值类型*：`pad`|`reflect`|`repeat`；*默认值*：`pad`；*动画性*：**有**
- {{SVGAttr("xlink:href")}}
  - : {{Deprecated_Header}}对另一个将被用作模板的 `<radialGradient>` 元素的 [\<IRI>](/zh-CN/docs/Web/SVG/Content_type#iri) 引用。*值类型*：[**\<IRI>**](/zh-CN/docs/Web/SVG/Content_type#iri)；*默认值*：none；*动画性*：**有**
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**